src/docfx/lib/path/FilePath.cs

```diff
@@ -33,6 +33,11 @@ internal class FilePath : IEquatable<FilePath>, IComparable<FilePath>
         /// </summary>
         public FileOrigin Origin { get; }
 
+        /// <summary>
+        /// Get the original path for splited files.
+        /// </summary>
+        public FilePath? OriginalPath { get; }
```

Comment:
Lets keep `FilePath` minimal as an identifier in wire format. For this task, if there is an `a/toc.yml` _fileMetadata_ entry, add a `a/_splitted/**` _fileMetadata_ entry is sufficient.